src/handlers/web3.js

```diff
@@ -350,12 +385,13 @@ export const getTransferTokenTransaction = async transaction => {
   );
   const recipient = await resolveNameOrAddress(transaction.to);
   const data = getDataForTokenTransfer(value, recipient);
+  const gasParams = getTransactionGasParams(transaction);
```

Comment:
we can remove the comment (lines 376-380) as it is outdated and will be TS'd shortly